src/main/external-resources/renderers/Panasonic-DMRBWT740.conf

```diff
@@ -0,0 +1,56 @@
+#----------------------------------------------------------------------------
+# Profile for Panasonic DMR-BWT740.
+# See DefaultRenderer.conf for descriptions of all the available options.
+#
+
+RendererName = Panasonic DMR
+RendererIcon = Panasonic-DMRBWT740.png
+
+# ============================================================================
+# This renderer has sent the following string/s:
+#
+# User-Agent: Panasonic MIL DLNA CP UPnP/1.0 DLNADOC/1.50
+# friendlyName=DMR-BWT740
+# udn=uuid:4D454930-0300-1000-8000-A81374478316
+# manufacturer=Panasonic
+# modelName=BD/DVD Recorder
+# modelNumber=DMR-BWT740
+# modelDescription=Panasonic BD/DVD Recorder
+# manufacturerURL=
+# modelURL=
+# ============================================================================
+# https://www.richersounds.com/static/manuals/DMRBWT740.pdf
+
+UpnpDetailsSearch = DMR-(B|P)WT
```

Comment:
@SubJunk Ok, i guess that i should be more general:
http://www.universalmediaserver.com/forum/viewtopic.php?f=9&t=9173&p=29618#p29618

I didn't even now all these german models...
@Nadahar I'll try to modify it to `DMR-(B|P)` but i don't what is the more clean/correct way to do that...